README.md

```diff
@@ -1,2 +1,51 @@
 # test
 
```

Comment:
Will it be a good idea to include the design of current test workflows? 
Maybe something like this [Prometheus test-infra](https://github.com/prometheus/test-infra/blob/master/prombench/README.md)?
I think it might be too much for this stage, so maybe in the future?

Reply:
I think we can defer until later on :+1: